### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by spike886**

### CODE REVIEW

**Comment by elbrujohalcon** [comment]
@spike886 don't merge the PRs yourself, get @sildur to review them, please :)

@sildur can you review this, please?

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (elbrujohalcon) [inline .editorconfig]: shouldn't this file be in the .gitignore? Isn't it just for a particular text editor?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,66 @@
-[![Stories in Ready](https://badge.waffle.io/inaka/react-component-inspector.png?label=ready&title=Ready)](https://waffle.io/inaka/react-component-inspector)
-# react-component-inspector
+[![NPM version][npm-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Dependency Status][daviddm-image]][daviddm-url] [![Code Climate][codeClimate-image]][codeClimate-url] [![Test Coverage][coverage-image]][coverage-url]
+[![Stories in Ready][waffle-image]][waffle-url] 
+
+React Component Inspector
+==========
+
+This add to you components a attribute, with the description of the whole component
```
Comment (elbrujohalcon) [inline README.md]: **React Component Inspector** adds an attribute to your component that will hold the description of the whole component.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,66 @@
...
+
+Installation
+------------
+
+Coming soon `npm install react-component-inspector`
+
+Usage
+-----
+
+In your main application file, require and call the module, you'll components
```
Comment (elbrujohalcon) [inline README.md]: ...call the module, then your components

File: README.md [hunk 4]
```diff
@@ -1,5 +1,66 @@
...
+will get a _componentInspector attribute with the description of itself
```
Comment (elbrujohalcon) [inline README.md]: ...with their own descrptions

or

...with the description of themselves

File: README.md [hunk 5]
```diff
@@ -1,5 +1,66 @@
...
+
+```js
+var inspector = require('react-component-inspector');
+if (ENV === 'development') inspector();
+```
+
+You probably don't want to call it if you're in production, and better
+yet, alias the module to nothing with webpack in production.
```
Comment (elbrujohalcon) [inline README.md]: yet, you should alias the module...

File: README.md [hunk 6]
```diff
@@ -1,5 +1,66 @@
...
+please use our public [hipchat room](https://www.hipchat.com/gpBpW3SsT).
+
+If you find any **bugs** or have a **problem** while using this library, please
+[open an issue](https://github.com/inaka/react-component-inspector/issues/new) in this repo
+(and a pull request if you want too :)).
+
+Author
+------
+
+Andrés Sebastián Gracía Scardilli
```
Comment (elbrujohalcon) [inline README.md]: you should link to your github user, like...
[spike886](github.com/spike886) - Andrés Sebastián García Scardilli

File: .editorconfig (hunk 1 above)
Comment (spike886) [inline .editorconfig]: This is not IDE specific config file. http://editorconfig.org/
The is made to define and maintain consistent coding styles, between developers

Comment (elbrujohalcon) [inline .editorconfig]: Are those rules checked automatically? In that case, which tool is used to check them?

Comment (spike886) [inline .editorconfig]: Is not a file to configure the style checker. It's a file to configure the
style used on the IDE. But it is not a configuration file for an specific
IDE. It's an IDE agnostic style configuration file.

Comment (elbrujohalcon) [inline .editorconfig]: Oh, [I see](editorconfig.org)

**Pull request merged**